core/src/main/java/com/datastrato/gravitino/connector/BaseCatalogPropertiesMetadata.java

```diff
@@ -15,6 +15,9 @@
 @Evolving
 public abstract class BaseCatalogPropertiesMetadata extends BasePropertiesMetadata {
 
+  public static final String CLOUD_NAME = "cloud.name";
+  public static final String CLOUD_REGION_CODE = "cloud.region-code";
```

Comment:
It is better to define these two reserved properties in Catalog interface. So users will know how to use it. Also don't forget to update the doc.